### ISSUE TITLE
About Inference

### ISSUE THREAD

**Issue by henrycjh** [Advocate99/DiffGesture#5]
Thanks for your great work! It seems that the inference code is running on the validation set. Is it possible to run on an arbitrary audio?

**Comment by Advocate99** [comment]
Hi. Thanks.
Yes, the uploaded code works on the validation set. For driving preposes with arbitrary audio, you can edit the testing code or the dataloader code. 
Since our model is trained on TED speech data, the arbitrary speech audio may perform well while other novel types of audio may not produce good performances considering the domain gap.
